metadata/src/specs.rs

```diff
@@ -372,9 +425,9 @@ impl<M, R> MessageSpecBuilder<Missing<state::Selector>, M, R> {
     }
 }
 
-impl<S, R> MessageSpecBuilder<S, Missing<state::Mutates>, R> {
+impl<S, P, R> MessageSpecBuilder<S, Missing<state::Mutates>, P, R> {
     /// Sets if the message is mutable, thus taking `&mut self` or not thus taking `&self`.
```

Comment:
Is this comment outdated? The function takes `self`.

Reply:
Yeah this is a builder pattern so we can construct the metadata of the ink! entity in question. So `self` is not what we are referring to with it.